airflow/providers/databricks/sensors/sql.py

```diff
@@ -0,0 +1,126 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+"""This module contains Databricks sensors."""
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any, Callable, Sequence
+
+from airflow.providers.common.sql.hooks.sql import fetch_all_handler
+from airflow.providers.databricks.hooks.databricks_sql import DatabricksSqlHook
+from airflow.sensors.base import BaseSensorOperator
+
+if TYPE_CHECKING:
+    from airflow.utils.context import Context
+
+
+class DatabricksSqlSensor(BaseSensorOperator):
+    """
+    Sensor to execute SQL statements on a Delta table via Databricks.
+
+        :param databricks_conn_id: Reference to :ref:`Databricks
+            connection id<howto/connection:databricks>` (templated), defaults to
+            DatabricksSqlHook.default_conn_name
+        :param http_path: Optional string specifying HTTP path of Databricks SQL Endpoint or cluster.
+            If not specified, it should be either specified in the Databricks connection's
+            extra parameters, or ``sql_endpoint_name`` must be specified.
+        :param sql_endpoint_name: Optional name of Databricks SQL Endpoint. If not specified, ``http_path``
+            must be provided as described above, defaults to None
+        :param session_configuration: An optional dictionary of Spark session parameters. If not specified,
+            it could be specified in the Databricks connection's extra parameters., defaults to None
+        :param http_headers: An optional list of (k, v) pairs
+            that will be set as HTTP headers on every request. (templated).
+        :param catalog: An optional initial catalog to use.
+            Requires DBR version 9.0+ (templated), defaults to ""
```

Comment:
It seems to default to `"hive_metastore"` not `""`

Reply:
Changed it to `""`